Form/Transformer/EntityToIdObjectTransformer.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Form\Transformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+use Doctrine\Common\Persistence\ObjectManager;
+
+/**
+ * Class EntityToIdObjectTransformer
+ *
+ * @author Marc Juchli <mail@marcjuch.li>
+ */
+class EntityToIdObjectTransformer implements DataTransformerInterface {
+
+    /**
+     * @var ObjectManager
+     */
+    private $om;
+
+    /**
+     * @var String
+     */
+    private $entityName;
+
+    /**
+     * @param ObjectManager $om
+     */
```

Comment:
The docblock should be either complete (it's currently missing the `$entityName` parameter) or should be omitted otherwise.